app/src/loginService.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+    angular
+        .module('webshopModule')
+        .service('loginService', LoginService);
+
+    function LoginService(api, $window) {
+
+        var session = {authenticated: false};
+
+        this.login = function (email, password) {
+            return api.login(email, password)
+                .then(function (token) {
+                    session.authenticated = true;
+                    $window.localStorage.setItem('token', token);
+                });
+        };
+
+        this.getToken = function () {
```

Comment:
Please rename this function to `getSession`, and store the token in the session (during initialization or login).